src/main/java/com/commercetools/helper/mapper/impl/payment/BasePaymentMapperImpl.java

```diff
@@ -253,6 +253,11 @@ protected Item createPaypalPlusItem(@Nonnull LocalizedString itemName, @Nonnull
         return createPaypalPlusItem(itemName, locales, dlipfq.getQuantity(), dlipfq.getDiscountedPrice().getValue());
     }
 
+    private Item plusSkuProductName(@Nonnull LineItem lineItem, @Nonnull List<Locale> locales, @Nonnull Item targetItem) {
+        return targetItem.setSku(lineItem.getVariant().getSku())
+                .setName(lineItem.getName().get(locales));
```

Comment:
Is it ensured that locale is never null?

Reply:
Yes. https://github.com/commercetools/commercetools-paypal-plus-integration/blob/aa0be5dbd8c5d1724ca641ba62582daa4bef1bde/src/main/java/com/commercetools/model/CtpPaymentWithCart.java#L123